### PR TITLE
Add Attributz

### PR DESCRIPTION
**Pull request by attributz**

### CODE REVIEW

**Comment by attributz** [comment]
@googlebot I signed it!

File: sources/attributz.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "id": "ATTRIBUTZ",
+    "name": "Attributz",
+    "categories": ["ADVERTISING", "ANALYTICS", "MARKETING", "TOOLS"],
+    "organization": "Attributz",
```
Comment (pfrisella) [inline sources/attributz.json:5]: ATTRIBUTZ

**Review by pfrisella — COMMENTED** [review]
Sources are not necessarily supposed to be tied to a connector. It is supposed to represent the source of data. If your connector is sourcing data from a Google Sheet, perhaps that would be a more appropriate source.  E.g. GOOGLE_SHEETS or CUSTOM_SPREADSHEET

**Comment by attributz** [comment]
@pfrisella makes sense, thank you

**Pull request closed without merging**